<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MyAddNew</class>
 <widget class="QWidget" name="MyAddNew">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_10">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="background">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QFrame" name="frame">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_4" stretch="2,5,1,1,1,1">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>50</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>100</number>
         </property>
         <item>
          <widget class="QLabel" name="label_up">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="styleSheet">
            <string notr="true">font: 87 15pt &quot;思源宋体 CN Heavy&quot;;
background-color: rgba(255, 255, 255, 175);</string>
           </property>
           <property name="text">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:16pt;&quot;&gt;元培地下电影院日记本&lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:16pt;&quot;&gt;YPUC LOGGER&lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:16pt;&quot;&gt;添加新片&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_7" stretch="4,3,3,4">
           <property name="spacing">
            <number>7</number>
           </property>
           <property name="leftMargin">
            <number>6</number>
           </property>
           <property name="topMargin">
            <number>6</number>
           </property>
           <property name="rightMargin">
            <number>6</number>
           </property>
           <property name="bottomMargin">
            <number>6</number>
           </property>
           <item>
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_3" stretch="5,1">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_8" stretch="1,3,1">
               <property name="spacing">
                <number>5</number>
               </property>
               <item>
                <spacer name="horizontalSpacer_3">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>40</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item>
                <widget class="QLabel" name="label_showposter">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="font">
                  <font>
                   <family>思源宋体 CN Heavy</family>
                   <pointsize>12</pointsize>
                   <weight>75</weight>
                   <bold>true</bold>
                  </font>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">background-color: rgba(255, 255, 255, 175);</string>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
               <item>
                <spacer name="horizontalSpacer_4">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>40</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_9" stretch="0">
               <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
                <widget class="QPushButton" name="figureChooseButton">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Minimum" vsizetype="Maximum">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="font">
                  <font>
                   <family>Agency FB</family>
                   <pointsize>10</pointsize>
                  </font>
                 </property>
                 <property name="cursor">
                  <cursorShape>ArrowCursor</cursorShape>
                 </property>
                 <property name="autoFillBackground">
                  <bool>true</bool>
                 </property>
                 <property name="styleSheet">
                  <string notr="true"/>
                 </property>
                 <property name="text">
                  <string>+ 点击以添加图片</string>
                 </property>
                 <property name="autoDefault">
                  <bool>false</bool>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout" stretch="1,1,1">
             <property name="spacing">
              <number>2</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="1,2">
               <property name="rightMargin">
                <number>3</number>
               </property>
               <item>
                <widget class="QLabel" name="displayTime">
                 <property name="text">
                  <string>     放映日期：</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QDateEdit" name="dateEdit">
                 <property name="dateTime">
                  <datetime>
                   <hour>0</hour>
                   <minute>0</minute>
                   <second>0</second>
                   <year>2022</year>
                   <month>9</month>
                   <day>14</day>
                  </datetime>
                 </property>
                 <property name="minimumDateTime">
                  <datetime>
                   <hour>0</hour>
                   <minute>0</minute>
                   <second>0</second>
                   <year>2022</year>
                   <month>6</month>
                   <day>1</day>
                  </datetime>
                 </property>
                 <property name="maximumDate">
                  <date>
                   <year>2025</year>
                   <month>7</month>
                   <day>1</day>
                  </date>
                 </property>
                 <property name="minimumDate">
                  <date>
                   <year>2022</year>
                   <month>6</month>
                   <day>1</day>
                  </date>
                 </property>
                 <property name="calendarPopup">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="1,2">
               <property name="rightMargin">
                <number>3</number>
               </property>
               <item>
                <widget class="QLabel" name="startTime">
                 <property name="text">
                  <string>     开始时间：</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QTimeEdit" name="timeEdit_start">
                 <property name="calendarPopup">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="1,2">
               <property name="rightMargin">
                <number>3</number>
               </property>
               <item>
                <widget class="QLabel" name="EndTime">
                 <property name="text">
                  <string>     结束时间：</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QTimeEdit" name="timeEdit_end">
                 <property name="calendarPopup">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,0,1,0,1,0,1,1">
           <item>
            <widget class="QLabel" name="blank1">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="movieName_label">
             <property name="text">
              <string>电影名：</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="nameInput"/>
           </item>
           <item>
            <widget class="QLabel" name="year">
             <property name="text">
              <string>    年份：</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="yearInput"/>
           </item>
           <item>
            <widget class="QLabel" name="region">
             <property name="text">
              <string>    地区：</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="regionInput"/>
           </item>
           <item>
            <widget class="QLabel" name="blank2">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,0,0,0,0,1">
           <item>
            <widget class="QLabel" name="blank4">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="director">
             <property name="text">
              <string>    导演：</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="directorInput"/>
           </item>
           <item>
            <widget class="QLabel" name="db_score">
             <property name="text">
              <string>    豆瓣评分：</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="dbscoreInput"/>
           </item>
           <item>
            <widget class="QLabel" name="blank3">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_6" stretch="3,1,10,3">
           <item>
            <widget class="QLabel" name="blank5">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="introduction">
             <property name="text">
              <string>简介：</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QTextEdit" name="introductionInput">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="blank6">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_11">
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="confirmButton">
             <property name="text">
              <string>确定</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="returnButton">
             <property name="text">
              <string>取消</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_2">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
